packages/dbgpt-serve/src/dbgpt_serve/observability/api/endpoints.py

```
@@ -0,0 +1,163 @@
+    return Result.succ(_dump(service.provider.model_usage(time_from, time_to)))
+
+
+@router.get("/traces")
+async def search_traces(
+    trace_id: Optional[str] = Query(default=None),
+    agent_name: Optional[str] = Query(default=None),
+    model_name: Optional[str] = Query(default=None),
+    conversation_id: Optional[str] = Query(default=None),
+    operation_name: Optional[str] = Query(default=None),
+    status: Optional[str] = Query(default=None),
+    start_time: Optional[datetime] = Query(default=None),
```

Ruff (0.16.2)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)